DNN Platform/Modules/DnnExportImport/Components/Services/PagesExportService.cs

```diff
@@ -2,2239 +2,2280 @@
+                        this.Result.AddLogEntry("Updated Tab", $"{otherTab.TabName} ({otherTab.TabPath})");
+                        this.totals.TotalTabs++;
+                        break;
+                    default:
+                        throw new ArgumentOutOfRangeException(this.importDto.CollisionResolution.ToString());
+                }
+            }
+            else
+            {
+                localTab = new TabInfo { PortalID = portalId };
+                SetTabData(localTab, otherTab);
+                localTab.StateID = this.GetLocalStateId(otherTab.StateID);
+                var parentId = this.IgnoreParentMatch ? otherTab.ParentId.GetValueOrDefault(Null.NullInteger) : TryFindLocalParentTabId(otherTab, exportedTabs, localTabs);
+                var checkPartial = false;
+                if (parentId == -1 && otherTab.ParentId > 0)
+                {
+                    if (!isParentPresent)
+                    {
+                        this.Result.AddLogEntry("Importing new tab skipped as its parent was not found", $"{otherTab.TabName} ({otherTab.TabPath})", ReportLevel.Warn);
+                        return;
+                    }
+
+                    checkPartial = true;
+                }
+
+                var tabType = Globals.GetURLType(otherTab.Url);
+                try
+                {
+                    localTab.ParentId = parentId;
+                    if (tabType == TabType.Url)
+                    {
+                        localTab.Url = otherTab.Url;
+                    }
+
+                    localTab.UniqueId = Guid.NewGuid();
+                    this.SetPartialImportSettings(otherTab, localTab);
+                    otherTab.LocalId = localTab.TabID = this.tabController.AddTab(localTab, false);
+                    DotNetNuke.Data.DataProvider.Instance().UpdateTabOrder(localTab.TabID, localTab.TabOrder, localTab.ParentId, Null.NullInteger);
+                    localTabs.Add(localTab);
+
+                    if (tabType == TabType.Tab && !referenceTabs.Contains(localTab.TabID))
+                    {
+                        referenceTabs.Add(localTab.TabID);
+                    }
+
+                    if (checkPartial)
+                    {
+                        this.CheckForPartialImportedTabs(otherTab);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    this.Result.AddLogEntry($"Importing tab '{otherTab.TabName}' exception", ex.Message, ReportLevel.Error);
+                    return;
+                }
+
+                this.UpdateTabChangers(localTab.TabID, createdBy, modifiedBy);
+
+                // this is not saved upon updating the tab
+                localTab.IsVisible = otherTab.IsVisible;
+                EntitiesController.Instance.SetTabSpecificData(localTab.TabID, false, localTab.IsVisible);
+
+                // tabController.UpdateTab(localTab); // to clear cache

```

Comment:
Should we remove commented out code ?

Reply:
We can, doesn't matter much to me either way.  For both of them you've called out here, there's at least a comment indicating the thinking somewhat, which might protect against re-making the same mistake.

Reply:
Ok, makes sense